docs/content/en/docs/reference/artifacts.md

```diff
@@ -30,19 +30,19 @@ However, see [Building node images]({{< relref "#building-node-images">}}) for i
 
```

Comment:
Could you also update the bundle numbers on image builder CLI URIs in [this](https://anywhere.eks.amazonaws.com/docs/reference/artifacts/#building-node-images) section?